jobs/kpi-forecasting/kpi_forecasting/configs/search_forecasting_model_performance.yaml

```diff
@@ -0,0 +1,6 @@
+input_config_list: ["search_forecasting_ad_clicks.yaml",  "search_forecasting_daily_active_users.yaml", "search_forecasting_search_count.yaml"]
```

Comment:
What are your thoughts on merging this with the current config files? I'd prefer having a `validation` section in each of those files

Reply:
I think the strongest argument not to do so is that the validation/performance is at the table level, and many config files aggregate up to make a single table.  But really that might be an argument for agglomerating  the individual configs so we have table-level configs rather than against merging the model performance configs with the existing ones

Reply:
If you like the idea of combining the existing config files so that each config file represents a single output table, maybe it makes sense to keep it as is for now and address the config files when we refactor?

Reply:
Yeah, I agree, the forecast config files should be combined into a single file for each data source. That makes sense, to keep it as-is until that config change is made.